card/lib/cardio.rb

```diff
@@ -23,10 +22,11 @@ def cache
     end
     
     def set_config config
-      @@config = config
-      @@root = @@config.root
-        
-      config.autoload_paths += Dir["#{Cardio.gem_root}/mod/*/lib/**/"]
+      @@config, @@root = config, config.root
+
+      config.autoload_paths += Dir["#{gem_root}/mod/*/lib/**/"]
+      config.autoload_paths += Dir["#{gem_root}/lib/**/"]
```

Comment:
I suppose one of us should investigate that.

One thing that I'm not clear on.  This doesn't mean that it loads all the matching files, but that it can load them with the auto-resolve constants stuff, right?